### PR TITLE
[WIP] Mix dep

### PR DESCRIPTION
**Pull request by sivsushruth**

### CODE REVIEW

**Comment by benoitc** [comment]
@sivsushruth can you add a task in travis to test this build? simply adding a line after rebar ct is done to launch a build with mix would be enough for now.

File: mix.exs [hunk 1]
```diff
@@ -52,22 +52,37 @@ defmodule Barrel.Mixfile do
...
     [
-      {:barrel_ex_logger, git: "https://github.com/barrel-db/barrel_ex_logger.git"},
+      {:lager_logger, "~> 1.0"},
       {:exrm, "~> 0.18.1"},
       {:barrel_nifs, path: "support/barrel_release_plugin"},
-      {:hooks,  "~> 1.1.1"},
-      {:mochicow, "~> 0.6.4"},
-      {:snappy, "~> 1.1"},
-      {:ucol, "~> 2.0"},
+      {:metrics, "~> 1.0.1", manager: :rebar, override: true},
```
Comment (josevalim) [inline mix.exs]: Shouldn't those be `manager: :rebar3` so we use latest rebar?

Comment (benoitc) [inline mix.exs]: rebar3 is the default and doesn't work. meanwhile @sivsushruth is working
on a minimal case

File: test/barrel_task_status_tests.erl [hunk 2]
```diff
@@ -16,7 +16,7 @@
 -module(barrel_task_status_tests).
 
 -include_lib("eunit/include/eunit.hrl").
--include("db.hrl").
+-include("../include/db.hrl").
```
Comment (benoitc) [inline test/barrel_task_status_tests.erl]: why is this needed? what was the error causing it?

Comment (sivsushruth) [inline test/barrel_task_status_tests.erl]: When tests are run via mix eunit or ct, tests are not able to access include folder directly, maybe need to set erl options.
